NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
The same text also says:

```Copyright notifications which have been relocated from source files (rather than removed) must be preserved in NOTICE.```

I think this is a simple and easy fix. it's concise and straight to the point.